posts/2022-06-19-attending-recruiting-fairs-as-an-engineer-b89.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Attending recruiting fairs as an engineer
+description: >
+  Over the years, I’ve attended many student recruitment fairs representing
+  Red Hat. Is this the best usage of an engineer’s expensive work hours?
+canonical: https://dev.to/tbrisker/attending-recruiting-fairs-as-an-engineer-b89
+date: 2022-06-19
+dir: ltr
+lang: en
+author:
+  name: Tomer Brisker
+  jobTitle: Principal Software Engineer
+tags:
+- hiring
+- recruiting
+- career
+- culture
+---
+
+![Cover image](./assets/cover_image.jpeg)
```

Comment:
Should have a descriptive alt text, instead of "cover image" which doesn't describe the contents of the image